nf_core/devcontainer/devcontainer.Dockerfile

```
@@ -1,8 +1,8 @@
-# Test build locally before making a PR
-#   docker build -t gitpod:test -f nf_core/gitpod/gitpod.Dockerfile .
+# Test build locally before making a PR (from project root directory)
+#   docker build . -t devcontainer:local -f nf_core/devcontainer/devcontainer.Dockerfile
 
 # See https://docs.renovatebot.com/docker/#digest-pinning for why a digest is used.
-FROM gitpod/workspace-base@sha256:7f35e401405c38ebc1185dfc3d6c066f73a524e8b86641453ea4690cc5c24fb6
+FROM gitpod/workspace-base:2025-05-14-07-50-25
```

```suggestion
FROM gitpod/workspace-base@sha256:7f35e401405c38ebc1185dfc3d6c066f73a524e8b86641453ea4690cc5c24fb6
```
this is done via our renovate-bot automation